frontend/www/js/omegaup/components/problem/FilterTags.vue

```diff
@@ -35,16 +35,16 @@ import VueTypeaheadBootstrap from 'vue-typeahead-bootstrap';
   },
 })
 export default class FilterTags extends Vue {
-  @Prop() tags!: string[];
   @Prop() publicTags!: string[];
+  @Prop({ default: () => [] }) tags!: string[];
+  @Prop({ default: () => [] }) selectedTags!: string[];
 
   T = T;
-  selectedTags: string[] = [];
+  currentSelectedTags = this.selectedTags;
 
   addOtherTag(tag: string): void {
     if (!this.tags.includes(tag)) {
```

Comment:
```suggestion
    if (!this.currentSelectedTags.includes(tag)) {
```

si no, si se selecciona el mismo tag dos veces, se va a agregar dos veces.